pkg/xds/server/server.go

```diff
@@ -246,6 +270,13 @@ func (s *server) process(stream stream, reqCh <-chan *v2.DiscoveryRequest, defau
 				return status.Errorf(codes.Unavailable, "empty request")
 			}
 
+			// node field in discovery request is delta-compressed
```

Comment:
What do you mean by delta-compressed?

Reply:
copied "as is" from https://github.com/envoyproxy/go-control-plane/blob/34c8be4/pkg/server/server.go#L282